packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -148,6 +148,7 @@ export class CheckpointRepoImpl implements CheckpointRepo {
   }
 
   registerLsig (networkName: string, name: string, info: LsigInfo): CheckpointRepo {
+    if (info.lsig.tag) { info.lsig.tag = Uint8Array.from(info.lsig.tag); }
```

Comment:
why are we doing this ?

Reply:
Previously tag was being decoded in `cp` as `{ type: 'buffer', value: <value> }`. But we needed it to be `Uint8Array`, example projects were failing otherwise (on my system). You can check on your system too. Let me know if we don't need this.

Reply:
ok, we need this. Please add a comment here.(for later understanding)

Reply:
can we do this before `registerLsig` ?

Reply:
ok

Reply:
Yes. How about moving this to [`getLsig`](https://github.com/scale-it/algorand-builder/blob/master/packages/algob/src/lib/lsig.ts#L15) ?

Reply:
yes that would be better or `mkDelegatedLsig` works too

Reply:
Ok. moving to `getLsig`

Reply:
examples are working on my system if we change tag to type `Buffer`. tested asa example

Reply:
** in types-algosdk

Reply:
update: got error with buffer.